### PR TITLE
Cloudforming AWS Step function

### PR DESCRIPTION
**Pull request by paulbrown1982**
AWS have enabled cloudforming step functions so I've updated to use that instead of the custom-made one.

cc @jacobwinch

### CODE REVIEW

File: cloudformation.yaml [hunk 1]
```diff
@@ -84,23 +84,67 @@ Resources:
     Properties:
       AliasName: !Sub alias/zuora-creditor-kms-${Stage}
       TargetKeyId: !Ref LambdaKMSKey
-    DependsOn: LambdaKMSKey
```
Comment (jacobwinch) [inline cloudformation.yaml]: Why is this dependency is no longer valid?

File: cloudformation.yaml [hunk 2]
```diff
@@ -84,23 +84,67 @@ Resources:
...
+          "StartAt": "FirstState",
+          "States": {
+            "FirstState": {
+              "Type": "Task",
+              "Resource": "arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:zuora-creditor-${Stage}",
+              "Next": "wait_using_seconds"
+            },
+            "wait_using_seconds": {
+              "Type": "Wait",
+              "Seconds": 60,
```
Comment (jacobwinch) [inline cloudformation.yaml]: Does the export always complete within 60 seconds? Could play it safe and wait for longer if there is no cost implication involved?

Comment (paulbrown1982) [inline cloudformation.yaml]: Typically the export is picking up only 3 invoices a day, which Zuora completes on average within 10 seconds, so I think we're safe for now. I'll keep monitoring it. Any we miss will just get picked up the following day.

File: cloudformation.yaml (hunk 1 above)
Comment (paulbrown1982) [inline cloudformation.yaml]: It was superfluous as there was no ambiguity or circular dependency.

**Pull request merged**